### DISCUSSION TITLE
Add toggle to lock topbar clock state

### DISCUSSION THREAD

**Discussion by yesseruser** [ppy/osu#30381] [Ideas] [answered]
On mobile, sometimes I accidentally click the clock instead of the notification button. I would like an option to lock the state of the clock, so clicking it does nothing.

**Comment by Joehuu** [comment]
I agree. I believe it's a set once and forget kind of thing that should be left in the settings.

Maybe it can be another overlay toggle that shows time-sensitive things like daily challenge, fanart/contest submission and voting, tournament registration, etc.

**Comment by yesseruser** [comment]
I am willing to open a PR for this.

**Reply by peppy** [reply]
Please don't.

**Comment by peppy** [comment, marked as answer]
I'd sooner just make it a settings overlay dropdown and remove clicking. But honestly I think it's fine as is personally and you can just tap again to get it back how you want it.
